packages/google_sign_in/google_sign_in_web/example/integration_test/people_test.dart

```diff
@@ -52,7 +52,6 @@ void main() {
       expect(user.id, expectedPersonId);
       expect(user.displayName, expectedPersonName);
       expect(user.photoUrl, expectedPersonPhoto);
-      expect(user.idToken, isNull);
```

Comment:
I thought we no longer did the People API fallback on the web? Maybe this whole test can be removed as well?

Reply:
I think I did indeed just forget to delete these files.